app/services/translation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Translation
+  def self.price_category(price)
+    price = price.at_xpath("category").try(:text)
+
+    case price
+    when "children"
+      "Kinder"
+    when "adult"
+      "Erwachsene"
+    when "group"
+      "Gruppen"
+    when "family"
+      "Familien"
+    when "senior"
+      "Senioren"
+    when "reduced"
+      "Ermäßigt"
+    when "discount", "other"
+      price.at_xpath("categorytext").try(:text)
```

Comment:
Bei Abkapselung müsste das hier dann auch als zweiter Parameter `fallback_price = ''` zum Beispiel reinkommen. So kann das `price.at_xpath("categorytext").try(:text)` vorher stattfinden.

Reply:
oder eben die Weiche in der vorherigen Methode :category_name_for_price von `poi_record` belassen

Reply:
@donni106 Verstehe Deinen Kommentar nicht so ganz. Kannst Du erklären.